.ci/Jenkinsfile

```diff
@@ -123,9 +124,19 @@ pipeline {
     cleanup {
       notifyBuildResult(downstreamJobs: itsDownstreamJobs)
     }
+    unsuccessful {
+      whenFalse(isPR()) {
+        doSlackSend()
+      }
+    }
```

Comment:
You can use

```
	      notifyBuildResult(downstreamJobs: itsDownstreamJobs, slackComment: true, slackChannel: "#${params.SLACK_CHANNEL}")
```

And then the slack message will be sent for broken builds with the below information:

![image](https://user-images.githubusercontent.com/2871786/95756079-35fb6a00-0c9d-11eb-9765-8d97bc5b533b.png)